### PR TITLE
added glob matching for nicknames

### PR DESCRIPTION
**Pull request by SPFabGerman**
I added the ability to use glob matches for the nicknames.
An example where this could be usefull, is for Bluetooth devices:
`--sink-nickname "bluez_sink.*.a2dp_sink":""`
(Normally the sinks for the Bluetooth Headsets contain the MAC-Adress, which means that every used device has to be specified manually.)

### CODE REVIEW

File: pulseaudio-control.bash [hunk 1]
```diff
@@ -63,13 +63,24 @@ function getNickname() {
         SINK_NICKNAME="${SINK_NICKNAMES[$sinkName/$portName]}"
     elif [ -n "$sinkName" ] && [ -n "${SINK_NICKNAMES[$sinkName]}" ]; then
         SINK_NICKNAME="${SINK_NICKNAMES[$sinkName]}"
-    elif [ -n "$sinkName" ] && [ -n "$SINK_NICKNAMES_PROP" ]; then
+    elif [ -n "$sinkName" ]; then
+        # No exact match could be found, try a Glob Match
+        for glob in "${!SINK_NICKNAMES[@]}"; do
+	    # shellcheck disable=SC2053 # Disable Shellcheck warning for Glob-Matching
```
Comment (marioortizmanero) [inline pulseaudio-control.bash]: Indentation is off here, seems like you're using a tab instead of spaces here

**Review by marioortizmanero — COMMENTED** [review]
Thanks a lot for the PR!

Very interesting feature indeed. Could you also add some documentation to the `--help` message and the README?

Also, that whole function is a mess (my bad, not yours). I should add comments to the `if`s or clean it up because it's hard to follow...

**Comment by SPFabGerman** [comment]
All done!
I also fixed the indentation. (Seems like my editors were setup wrong and the single tab slipped through. The old problem with tabs and spaces, haha)

**Comment by marioortizmanero** [comment]
Great. Before merging this, do you think you could do the same for the blacklisting feature? It would surely be very helpful there too.

**Comment by SPFabGerman** [comment]
Done. Implementing the Glob Matching for the Backlist was super easy, because it already uses a loop.
(I also found another tab, that I just got fixed by my formatter.)

**Comment by marioortizmanero** [comment]
Thanks again! Merging, I'll release a new version soon.
